takari-lifecycle-plugin/pom.xml

```diff
@@ -21,7 +21,7 @@
 
   <!-- not inherited, IIRC -->
   <prerequisites>
-    <maven>[3.3.9,)</maven>
+    <maven>[3.6.3,)</maven>
```

Comment:
Can you confirm your intention is to desupport all earlier Maven versions with this PR?

Reply:
Correct. I leave it to you whether this justifies or warrants a significant version increase.